.github/workflows/build-cli.yml

```diff
@@ -284,11 +284,11 @@ jobs:
               chmod +x build.sh
               # Set Windows build environment and run build script
               GOOS=windows GOARCH=amd64 ./build.sh
-              
+
               # Move the built binary to the expected location (inside container)
               mkdir -p ../target/x86_64-pc-windows-gnu/release
               mv temporal-service.exe ../target/x86_64-pc-windows-gnu/release/temporal-service.exe
-              
+
```

Comment:
Gah, sorry – I have automatic linting on, and it linted all the changed files from the merge. Let me know if I should revert the whitespace issues.